tsconfig.json

```diff
@@ -8,10 +8,10 @@
     "reactNamespace": "React",
     "module": "ESNext",
     "moduleResolution": "Node",
+    "baseUrl": ".",
     "paths": {
-      "~/*": [
-        "/*"
-      ]
+      "~/*": ["*"],
+      "@divriots/dockit-core/*": ["*"]
```

Comment:
This is kinda nice actually, means we can use "consumer code" in our stories, making it more "copyable". We should do that for starter kits like simba as well.

Reply:
I'm preaching this idea for month and fixed Backligh code to be able to do that at some point :) so yes, go for it!